api/src/main/java/io/strimzi/api/kafka/model/kafka/KafkaResources.java

```diff
@@ -289,8 +289,8 @@ public static String entityOperatorDeploymentName(String clusterName) {
      * Returns the name of the Entity Operator {@code Secret} for a {@code Kafka} cluster of the given name.
      * This {@code Secret} will only exist if {@code Kafka.spec.entityOperator} is configured in the
      * {@code Kafka} resource with the given name.
-     *
-     * This secret is not used anymore and is deprecated. This method will be removed in the future.
+     * 
+     * <p>This secret is not used anymore and is deprecated. This method will be removed in the future.</p>
```

Comment:
I don't think this makes sense. We do not use this in most of the code, we do not really care about the generated Javadocs. And if we would want to use the HTML tags, it would not make much sense to use it only in half of the comment.

Reply:
There was a warning, that's why I changed that. Removed, but I think using a bit of HTML is fine in Javadoc, as long as it doesn't hurt readability.

Reply:
We do not have to follow every warning from an IDE. Also, the warnings are configurable. So a _warning from IDE_ is not a universal source of truth to do something. Empty lines in the Javadoc help to make things more readable when there make sense. And TBH that is the main reason why we have the Javadocs, so we should focus on its readability and keep the empty lines as they are.